04_setup_ironic.sh

```diff
@@ -67,6 +69,8 @@ fi
 # Create pod
 sudo podman pod create -n ironic-pod 
 
+IRONIC_RHCOS_DOWNLOADER_IMAGE=$(oc adm release info --registry-config $REGISTRY_AUTH_FILE $OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE --image-for=ironic-rhcos-downloader)
+
```

Comment:
CI seems to be failing on this like:

```
error: info expects one argument, or a connection to an OpenShift 4.x server: invalid configuration: Missing or incomplete configuration info.  Please login or point to an existing, complete config file:
```

Reply:
Y

> CI seems to be failing on this like:
> 
> ```
> error: info expects one argument, or a connection to an OpenShift 4.x server: invalid configuration: Missing or incomplete configuration info.  Please login or point to an existing, complete config file:
> ```

ya, I just noticed this this morning, I'm testing a new version at the moment, should be up soon.